kolibri/core/assets/src/utils/minimumBrowserRequirements.js

```diff
@@ -1,16 +1,30 @@
+import isUndefined from 'lodash/isUndefined';
+import browsers from 'browserslist-config-kolibri';
 import { browser, passesRequirements } from './browserInfo';
 import plugin_data from 'plugin_data';
 
-const minimumBrowserRequirements = {
-  IE: {
-    major: 11,
-  },
-  Android: {
-    major: 4,
-    minor: 0,
-    patch: 2,
-  },
-};
+const minimumBrowserRequirements = {};
+
+const browserRegex = /^([a-zA-Z]+) ([><=]+) (\d+)(?:\.(\d+))?(?:\.(\d+))?$/;
+
+for (const browser of browsers) {
+  const [name, sign, major, minor, patch] = browserRegex.exec(browser).slice(1);
+  // This only supports > and >=, but that's all we need.
+  const addOne = sign === '>' ? 1 : 0;
```

Comment:
`sign` here means literally `>` or `>=`?

Reply:
Yes, I hated this, and only resorted to this to avoid about 15 nested if statements. Will think about how to rework for clarity.